packages/element-templates-json-schema-shared/src/defs/base-properties.json

```diff
@@ -144,6 +169,18 @@
           }
         }
       },
+      "feel": {
```

Comment:
So we will support the `feel` property in both element templates worlds?

Reply:
If we only support it for `zeebe`, we could handle it the same as with `optional`: https://github.com/camunda/element-templates-json-schema/blob/master/packages/zeebe-element-templates-json-schema/src/defs/properties.json#L252

Maybe the rule could also be the same: https://github.com/camunda/element-templates-json-schema/blob/master/packages/zeebe-element-templates-json-schema/src/defs/properties.json#L128

Reply:
That looks promising, thanks for the hint!

As for where we can use it: We currently only use it in the zeebe properties panel. The implementation is currently agnostic of target platform, but that might change soon. I'll move it into the zeebe properties just to be safe

Reply:
👍